lookup.php

```diff
@@ -4,11 +4,12 @@
 	if (!isset($IP)) {
 		$Result = null;
 	} else {
-		if (function_exists('filter_var')) {
```

Comment:
What about validation of user input data?

Reply:
I don't understand about "user" input data...
Script is requesting "IP" info so you can't have nothing else?
And this function didn't worked for me (PHP 7.3.7) when there were IPv6 addresses (brackets issue).

Reply:
Some one can send not only IP to this API.
What if place this check after strip [ ] ?

Reply:
Instead of checking "filter_var" function availability, we can add a php.version requirement (PHP 5.2.0+).
But I don't think many people still use very very old PHP version below 5.2.0 so we can remove "function_exists"?

Reply:
Ok, lets set min PHP ver.

Reply:
Yes, LGTM!